xonsh/main.py

```diff
@@ -191,6 +191,13 @@ def parser():
         action="store_true",
         default=False,
     )
+    p.add_argument(
+        "--no-env",
+        help="Do not load parent environment variables.",
+        dest="noenv",
+        action="store_true",
```

Comment:
the double-negation gets confusing, probably better to do this:
```suggestion
        dest="env",
        action="store_false",
```